sale_commission/__openerp__.py

```diff
@@ -22,7 +22,7 @@
 ##############################################################################
 {
     'name': 'Sales commissions',
-    'version': '2.0',
+    'version': '2.1',
```

Comment:
Please use OCA versioning according to https://github.com/OCA/maintainer-tools/blob/master/CONTRIBUTING.md